docs/administration-guide/gaffer-config/graph-metadata.md

```diff
@@ -0,0 +1,97 @@
+# Graph Metadata Configuration
+
+The graph configuration file is a JSON file that configures few bits of the
+Gaffer graph. Primarily it is used to set the name and description along with
+any additional hooks to run before an operation chain e.g. to impose limits on
+max results etc. For the example a simple graph configuration file might look
+like:
+
+```json title="graphConfig.json"
+{
+    "graphId": "ExampleGraph",
+    "description": "An example graph"
+}
+```
+
+## Configuring a Standard Deployment
+
+To change any of the values for a standard Gaffer deployment all thats needed
+is to configure the JSON file for the `graphConfig`. The key value pairs in
+the file can then be configured as you wished and upon restarting the graph
```

Comment:
Typo and I'm not sure about this statement. If you are using an Accumulo store and you update the name of the graph like this, it will cause a new table to be used in Accumulo and any existing data would be lost.

This is mentioned in the section below. It might be possible to restructure this page so these sections can be read together.